src/components/templates/PosterPage/PosterPage.tsx

```diff
@@ -18,6 +18,7 @@ import { PosterPageControl } from "./components/PosterPageControl";
 import { PosterPageSettingsControl } from "./components/PosterPageSettingsControl";
 
 import { usePosterVideo } from "./usePosterVideo";
+import { ShareModal } from "./components/ShareModal";
```

Comment:
Keep imports from the same paths together.

```suggestion
import { PosterPageSettingsControl } from "./components/PosterPageSettingsControl";
import { ShareModal } from "./components/ShareModal";

import { usePosterVideo } from "./usePosterVideo";
```

Reply:
https://github.com/sparkletown/sparkle/pull/1396/commits/a6a886bae3d491ba531c9f7bf30a4b617c799724